renormalizer/mps/matrix.py

```diff
@@ -117,26 +111,26 @@ def check_lortho(self, rtol=1e-5, atol=1e-8):
         """
         check L-orthogonal
         """
-        tensm = self.array.reshape([np.prod(self.shape[:-1]), self.shape[-1]])
-        s = xp.dot(tensm.T.conj(), tensm)
-        return allclose(s, xp.eye(s.shape[0]), rtol=rtol, atol=atol)
+        tensm = asxp(self.array.reshape([np.prod(self.shape[:-1]), self.shape[-1]]))
+        s = tensm.T.conj() @ tensm
+        return xp.allclose(s, np.eye(s.shape[0]), rtol=rtol, atol=atol)
 
     def check_rortho(self, rtol=1e-5, atol=1e-8):
         """
         check R-orthogonal
         """
-        tensm = self.array.reshape([self.shape[0], np.prod(self.shape[1:])])
-        s = xp.dot(tensm, tensm.T.conj())
-        return allclose(s, xp.eye(s.shape[0]), rtol=rtol, atol=atol)
+        tensm = asxp(self.array.reshape([self.shape[0], np.prod(self.shape[1:])]))
+        s = tensm @ tensm.T.conj()
+        return xp.allclose(s, np.eye(s.shape[0]), rtol=rtol, atol=atol)
```

Comment:
Should this `np.eye` be replaced with `xp.eye`?

Reply:
makes sense to me...